app/gui2/src/stores/graph/__tests__/graphDatabase.test.ts

```diff
@@ -94,6 +95,7 @@ test('Reading graph from definition', () => {
 
   // Commented the connection from input node, as we don't support them yet.
   expect(Array.from(db.connections.allForward(), ([key]) => key)).toEqual([
+    id('parameter'),
     id('node1Id'),
     id('node2Id'),
   ])
```

Comment:
Please uncomment the comented expects below and remove the comment here.

Reply:
Done